recipe/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.pyc
+
+build_artifacts
```

Comment:
There are also some hidden files that would need to be removed.

Reply:
(from the `recipe` directory that is)

Reply:
@jakirkham my bad - should be good now.